src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FruitTransaction {
```

Comment:
Don't begin class implementation with an empty line. Please remove the redundant empty line to adhere to the checklist.